pynitrokey/cli/start.py

```diff
@@ -93,14 +95,81 @@ def rng(count, raw, quiet):
                 print(challenge.hex())
 
 
+class CardRemovedGPGAgentException(RuntimeWarning):
+    pass
+
+
+def gpg_agent_set_identity(identity: int):
+    from pexpect import run
```

Comment:
I think we don’t have `pexpect` as a dependency so far.  We should either use `subprocess.run` instead or add `pexpect` to `pyproject.toml`.